.github/workflows/ci.yml

```diff
@@ -85,4 +85,4 @@ jobs:
           distribution: 'temurin'
           java-version: 8
       - name: Run Tests with JDK 8
-        run: ./mvnw install -T1C -B -ntp -fae
+        run: ./mvnw install -T1C -B -ntp -faegit branch --delete <branch>
```

Comment:
why add `git branch --delete <branch>` ?

Reply:
Sorry for the paste mistake